engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql

```diff
@@ -18,3 +18,10 @@
 --;
 -- Schema upgrade from 4.14.0.0 to 4.15.0.0
 --;
+
+
+-- Add XenServer 8.1 hypervisor capabilities
+INSERT IGNORE INTO `cloud`.`hypervisor_capabilities`(uuid, hypervisor_type, hypervisor_version, max_guests_limit, max_data_volumes_limit, max_hosts_per_cluster, storage_motion_supported) values (UUID(), 'XenServer', '8.1.0', 1000, 253, 64, 1);
```

Comment:
Since version 8.0 Citrix has changed XenServer to "Citrix Hypervisor", you can find the name change [here](https://docs.citrix.com/en-us/citrix-virtual-apps-desktops/new-names-numbers.html). The inserts should be updated to have the new names.

Reply:
@Sjnas Adding that support with changed product name will require many more exhaustive changes. New product name must be supported in schema, UI and code as well for that.
For now, with this PR I'll just target XCP-ng 8.x support